sql/modules/Person.sql

```diff
@@ -14,6 +14,16 @@ $$ LANGUAGE SQL;
 COMMENT ON FUNCTION person__get_my_entity_id() IS
 $$ Returns the entity_id of the current, logged in user.$$;
 
+CREATE OR REPLACE FUNCTION person__get_my_id() RETURNS INT AS
```

Comment:
This one causes the blacklist test to fail: there's a new function signature now, which needs to be added to the black/whitelist.